### PR TITLE
Fixing small bugs.

### PR DESCRIPTION
**Pull request by xubonnie**
I made 3 changes:

1) Add picture of graph visualization to README file. When people come across this repository, it's a nice easy way for them to determine if this is something they're interested in. It saves a lot of their time since it avoids having to make a bunch of clicks and navigate too much.

2) Fixed maps referring issue, from line `multistreamDictionary.get("TRANSPORT").get("PACKET_SENT")`, for example. If you'd get this error: `Uncaught TypeError: Cannot read property 'get' of undefined` if TRANSPORT wasn't the category, or PACKET_SENT wasn't the event. 

3) Fixed issue where `packetsSentSizeLUT` would error because it's an empty array (so for example, if there are no packet sent events in the qlog). Line of code is: `scatter.set("ymin", (packetsSentSizeLUT[0].from < 5000) ? 0 : packetsSentSizeLUT[0].from );`.

### CODE REVIEW

File: standalone/graph.js [hunk 1]
```diff
@@ -102,9 +101,12 @@ function drawGraph(qlog, settings){
 			// - we need to keep track of which packets contribute which data, so we properly draw the ACKs and LOST events 
 			
 			// - so we create: big sparse array of sent packets with packet number as index, data contains the amount of data it covers 
-			
-			let packetsSent = multistreamDictionary.get("TRANSPORT").get("PACKET_SENT") || []; // || [] defaults to an empty array if there are no events of that type present in the log
-			let packetSentList = [];
+			let packetsSent = [];
+		   let packetSentList = [];
```
Comment (rmarx) [inline standalone/graph.js]: Could you add another tab before this to align it with the rest?

File: standalone/graph.js [hunk 2]
```diff
@@ -128,70 +130,77 @@ function drawGraph(qlog, settings){
...
-						// ackedNr will be the ACKed packet number of one of our SENT packets here 
-						for( let ackedNr = from; ackedNr <= to; ++ackedNr ){
-							// find the originally sent packet 
-							let sentPacket = packetSentList[ ackedNr ]; 
-							if( !sentPacket ){
-								console.error("Packet was ACKed that we didn't send... ignoring", ackedNr, frame, packet);
-								continue;
-							}
-                            
+			let packetsReceived =[];
```
Comment (rmarx) [inline standalone/graph.js:131]: similar remarks here: did you use the same indentation method as the rest of the file? Looks garbled, at least on github.

**Comment by rmarx** [comment]
Thank you for the changes! The code in itself looks good, but at least on Github the indentations look very garbled. Did you switch to tabs vs spaces or 4 vs 2 spaces or something? 

If you could update that, that would be great :)

File: standalone/graph.js (hunk 2 above)
Comment (xubonnie) [reply standalone/graph.js:131]: I'm not sure why it looks so strange on Github, it looks fine to me on my IDE. 
image

I followed the same indentation method.

**Comment by xubonnie** [comment]
Rebased. Not sure why on Github the indentation is so off :(

**Pull request merged**